content/projects/tdd/password-checker/part1/_index.md

```diff
@@ -142,6 +142,17 @@ Please don't re-implement the same check in two different places. Good code is D
 - [JS Errors](https://www.w3schools.com/js/js_errors.asp)
 - [Errors and Jasmine](https://stackoverflow.com/questions/4144686/how-to-write-a-test-which-expects-an-error-to-be-thrown-in-jasmine)
 
+### DRY code for JS
+
+A good way to practice DRY in this project is to create an object(s) that stores your password validators and then use the object throughout your code.
```

Comment:
I think learners might think that they can create more than one object of the same structure. its better to say an object, instead of objects so that they create an object and include all error strings and their corresponding regex in the same object.